.github/workflows/master.yml

```diff
@@ -32,7 +32,7 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     timeout-minutes: 30
-    if: github.event_name != 'pull_request' && github.event.pull_request.draft == false
+    if: github.event_name != 'pull_request' || github.event.pull_request.draft == false
```

Comment:
it seems like lint will always run except on draft PRs? does that mean we run the lint command on other events (like PR comments etc) as well?

Reply:
Nope, this workflow only runs on PRs and pushes to master or release branches (e.g. branch-2.8).

Reply:
The `on` filed on this file looks like this:

```
on:
  pull_request:
  push:
    branches:
      - master
      - branch-[0-9]+.[0-9]+
```

Reply:
Ah okay makes sense, thanks!